src/gui/transferlistmodel.h

```diff
@@ -30,15 +30,59 @@
 #ifndef TRANSFERLISTMODEL_H
 #define TRANSFERLISTMODEL_H
 
+#include "base/bittorrent/torrenthandle.h"
+
 #include <QAbstractListModel>
 #include <QList>
+#include <QWidget>
 
 namespace BitTorrent
 {
     class InfoHash;
     class TorrentHandle;
 }
 
+// convenience class to read text colors from QSS since QSS doesn't work with QObject
+class TransferListTextColors : public QWidget
+{
+    Q_OBJECT
+    Q_DISABLE_COPY(TransferListTextColors)
+
+public:
+    TransferListTextColors();
+
+    Q_PROPERTY(QColor unknown MEMBER m_unknown)
```

Comment:
Are those required to be `public`?

Reply:
Well making it private doesn't affect anything but doesn't property should always be public. As they are already modifying outside the class `setProperty`?

Reply:
This look otherwise: https://doc.qt.io/qt-5/properties.html#a-simple-example
IMO if it can be `private` then it shouldn't be `public`.

Reply:
done